runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/actions/excel.ts

```diff
@@ -33,6 +33,7 @@ const excelActionsTranslations = {
     'Process.Details.Modeler.Actions.Excel.RenameWorksheet.Label': 'Rename worksheet',
     'Process.Details.Modeler.Actions.Excel.RenameWorksheet.NewName': 'New name',
     'Process.Details.Modeler.Actions.Excel.RenameWorksheet.Worksheet.Info': 'Leaving the field empty will change the name of active worksheet',
+    'Process.Details.Modeler.Actions.Excel.WorksheetExists.Label': 'Does worksheet exist',
```

Comment:
I would name it Check if worksheet exists. I know it is a long name, but follows the convention of "do something".